docs/presto-admin-commands.rst

```diff
@@ -331,6 +331,29 @@ once, you could run ::
     sudo ./presto-admin connector add
     sudo ./presto-admin server restart
 
+Adding a Custom Connector
+--------------------------
+Not all connectors are available in Presto by default. For non default
```

Comment:
Just say "In order to add a custom connector not included with Presto, the jar must be ..."